packages/authentication/package.json

```diff
@@ -6,18 +6,18 @@
     "node": ">=6"
   },
   "scripts": {
-    "acceptance": "lb-dist mocha --opts node_modules/@loopback/build/mocha.ts.opts 'DIST/test/acceptance/**/*.js'",
+    "acceptance": "lb-mocha 'DIST/test/acceptance/**/*.js'",
```

Comment:
Please change single quotes to double quotes while you are changing test-related scripts in this file. (See my other comments below for more details.)